### ISSUE TITLE
Training on customized LLM loss got sort of high

### ISSUE THREAD

**Issue by lucasjinreal** [dvlab-research/MGM#28]
Hi, am using Qwen2 4b as LLM training the model, the template solving way I previous tried on LLaVa works OK, and no warnings appeared.

But when pretrain on minigemini, got unexpected loss result:

```
{'loss': 1.9597, 'learning_rate': 0.0007436649460805199, 'epoch': 0.36}                                                                                                                                                            
{'loss': 1.8565, 'learning_rate': 0.0007435934693368483, 'epoch': 0.36}                                                                                                                                                            
{'loss': 2.0595, 'learning_rate': 0.0007435219860653267, 'epoch': 0.36}                                                                                                                                                            
{'loss': 1.9304, 'learning_rate': 0.0007434504962678705, 'epoch': 0.36}                                                                                                                                                            
{'loss': 1.9705, 'learning_rate': 0.0007433789999463957, 'epoch': 0.36}                                                                                                                                                            
{'loss': 1.9536, 'learning_rate': 0.000743307497102818, 'epoch': 0.36}                                                                                                                                                             
{'loss': 1.9405, 'learning_rate': 0.0007432359877390538, 'epoch': 0.36}                                                                                                                                                            
{'loss': 1.8501, 'learning_rate': 0.0007431644718570192, 'epoch': 0.36}                                                                                                                                                            
{'loss': 1.8722, 'learning_rate': 0.000743092949458631, 'epoch': 0.36}                                                                                                                                                             
{'loss': 1.6984, 'learning_rate': 0.0007430214205458056, 'epoch': 0.36}                                                                                                                                                            
{'loss': 1.8309, 'learning_rate': 0.0007429498851204598, 'epoch': 0.36}                                                                                                                                                            
{'loss': 1.8681, 'learning_rate': 0.0007428783431845109, 'epoch': 0.36}                                                                                                                                                            
{'loss': 1.9383, 'learning_rate': 0.0007428067947398757, 'epoch': 0.36}                                                                                                                                                            
{'loss': 1.8059, 'learning_rate': 0.000742735239788472, 'epoch': 0.36}                                                                                                                                                             
{'loss': 1.8016, 'learning_rate': 0.0007426636783322172, 'epoch': 0.36}                                                                                                                                                            
{'loss': 2.0427, 'learning_rate': 0.0007425921103730288, 'epoch': 0.36}                                                                                                                                                            
{'loss': 1.899, 'learning_rate': 0.0007425205359128248, 'epoch': 0.36}  
```

Looks like stucked at ~1.8, no longer decrease, what could be the reason for this? (the data all same)

(I tried using both chat template in pretrain and finetune, same as mini does)

**Comment by yanwei-li** [comment]
Hi, I don't think the training loss can really reflect the model performance. And 1.8 is also within the normal pretrain loss range in my mind. Maybe you can check the final result, and try to use different LR if the training process is not stable.

**Comment by lucasjinreal** [comment]
@yanwei-li  thanks for your reply. However, I am using lava pretrain can get a ~0.8 minimal loss. Does using Gemma get also ~1.8 when pretrain? Can not get your loss curve so am just afraid it might boomed in pretrain stage.

**Comment by yanwei-li** [comment]
I check the loss of the Gemma model. In the pretraining stage, it converges to about 2.0. In the instruction tuning stage, it converges to about 1.0. Hope this info could help you.

**Comment by lucasjinreal** [comment]
@yanwei-li  Oh, so looks like my loss are normal

**Comment by Sanster** [comment]
> I check the loss of the Gemma model. In the pretraining stage, it converges to about 2.0. In the instruction tuning stage, it converges to about 1.0. Hope this info could help you.

@yanwei-li Can you share the loss when using other models (for example, Vicuna-7B-v1.5/Vicuna-13B-v1.5)? I am currently using qwen-14b as the base model, and the loss during the pre-training stage is also around 2.0.

**Comment by yanwei-li** [comment]
Thanks for your question. Please reopen it if have the same issue.
